packages/sol-tracing-utils/src/artifact_adapters/sol_compiler_artifact_adapter.ts

```diff
@@ -43,9 +43,14 @@ export class SolCompilerArtifactAdapter extends AbstractArtifactAdapter {
                 logUtils.warn(`${artifactFileName} doesn't contain bytecode. Skipping...`);
                 continue;
             }
-            let sources = _.keys(artifact.sources);
-            sources = _.map(sources, relativeFilePath => path.resolve(this._sourcesPath, relativeFilePath));
-            const sourceCodes = _.map(sources, (source: string) => fs.readFileSync(source).toString());
+            const sources: { [sourceId: number]: string } = {};
+            const sourceCodes: { [sourceId: number]: string } = {};
```

Comment:
Can we move this in-line type declaration somewhere so we don't need to re-declare it. DRY.